src/Tribe/Views/V2/Service_Provider.php

```diff
@@ -46,6 +46,9 @@ public function register() {
 
 		// Since the View main class will act as a DI container itself let's provide it with the global container.
 		View::set_container( $this->container );
+
+		// @todo: remove this when we hydrate the month view with data and we use the correct template tags.
+		require_once tribe( 'tec.main' )->plugin_path . 'src/Tribe/Views/V2/month-view-demo-template-tags.php';
```

Comment:
Can we move this into the template files for month view? I makes it easier to not forget.